docs/src/running-validator/validator-reqs.md

```diff
@@ -63,7 +63,13 @@ Solana.
 
 In fact, the mainnet-beta validators operated by the team are currently
 (Mar. 2021) run on GCE `n2-standard-32` (32 vCPUs, 128 GB memory) instances with
-2048 GB SSD for operational convenience.
+2048 GB SSD for operational convenience. If you prefer AWS then (based on tests ran by vendor 
+later on May 2023) `r6a.8xlarge` (32 vCPU, 256 GB memory) instances work well with 
+three EBS gp3 volumes for better perfromance:
+
+- Root: EBS gp3 100GB, 3K IOPS, 125 MB/s throughput;
+- Accounts volume: EBS gp3 1TB, 5K IOPS, 700 MB/s throughput; 
+- Data volume: EBS gp3 10K IOPS, 700 MB/s throughput).
```

Comment:
i think the right parenthesis is not needed

Reply:
I thought it should be easier to read this way, with clear separation for each property of the volume config.

Reply:
I actually liked the previous version as well with individual lines for each drive. The comment was saying that there was an unmatched right parenthesis. Ie, change:
```
- Data volume: EBS gp3 10K IOPS, 700 MB/s throughput).
```
to
```
- Data volume: EBS gp3 10K IOPS, 700 MB/s throughput
```